### ISSUE TITLE
Analysis

### ISSUE THREAD

**Issue by mantono** [mantono/DuplicateSearcher#11]
**What should be done?**
Analysis of all issues comparing them to each other (not neccessarily n²)

**Input:** A set of issues
**Output:** All issues which are duplicates of another issue in the input set.

**Issues it depends on:** #4

**Comment by mantono** [comment]
**Alternatives**
1. Jaccard
2. Inverse Document Frequency
3. Term Frequency & Inverse Document Frequency
4. Vector Space Model (using either one of the above) with Cosine

**Comment by mantono** [comment]
Found [this method](http://docs.oracle.com/javase/8/docs/api/java/lang/Math.html#log1p-double-) by accident, could maybe be of interest for our Analyzer or any of our FrequencyCounter classes.
